.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1,2 +1,2 @@
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.2/apache-maven-3.6.2-bin.zip
-wrapperUrl=https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/0.5.5/maven-wrapper-0.5.5.jar
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
+wrapperUrl=https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/0.5.6/maven-wrapper-0.5.6.jar
```

Comment:
Due to bug in wrapper, this doesn't work on Windows + Bash (or maybe zsh) if the file ends with newline (you may have noticed me removing the newline in some PRs)

Reply:
ah thanks. is this already fixed or maybe going to upstream? https://github.com/takari/maven-wrapper/commit/7039b05cfec0e8a729d6d84670433912e00c6aa5

I'll remove the newline from here anyway. thanks for the tip.